subprojects/docs/src/docs/release/notes.md

```diff
@@ -11,6 +11,19 @@ you don't need to hardcode it into the build script - you can just run `gradle r
 
 This release of Gradle includes a [new Publishing Overview chapter](/userguide/publishing_overview.html) in the user manual and updates throughout the documentation regarding publishing artifacts using Maven and Ivy.  
 
+### Improved dependency insight report
+
+The dependency insight report is the distant ancestor of [build scans](https://scans.gradle.com) and helps you diagnose dependency management problems locally.
+This release of Gradle implements several improvements:
```

Comment:
Would this section be better placed in the promoted feature part? And here have a shorter write up?

Reply:
I don't think so, this is visible on top, so it's more natural to have more details first, rather than here.